src/fides/api/ops/schemas/privacy_request.py

```diff
@@ -60,6 +60,8 @@ class Consent(BaseSchema):
     data_use: str
     data_use_description: Optional[str] = None
     opt_in: bool
+    has_gpc_flag: Optional[bool] = False
+    conflicts_with_gpc: Optional[bool] = False
```

Comment:
Since these can't be `None` I don't think you need `Optional`. You can still default to `False` and they don't have to be sent. 

```suggestion
    has_gpc_flag: bool = False
    conflicts_with_gpc: bool = False
```